App.js

```diff
@@ -6,109 +6,403 @@
+
+      <Text style={styles.estiloTextoResultado}>{this.state.textoResultado}</Text>
+
+      </View>
+
+      <View style={styles.divisorPantallaBotones}>
+      <TouchableOpacity
+              style={styles.estiloBotonesBorrado}
+              onPress={()=>this.LimpiarPantalla()}  
+              >
+             <Text style={styles.estiloTextoBotonBorrado}>Delete</Text>
+           </TouchableOpacity>
+  
+
+        <TouchableOpacity
+              style={styles.estiloBotonesBorrado}
+              onPress={()=>this.BorrarUltimoCaracter()}
+              >
+             <Text style={styles.estiloTextoBotonBorrado}>&#60;&#60;</Text>
+           </TouchableOpacity>
+      </View>
+
+      <View style={styles.containerBotones}>
+       
+          <View style={styles.columnaBotones}>
+
+            {columnasNumeros[0]}
+
+            <TouchableOpacity
+              style={styles.estiloBotonDecimal_Signo}
+              onPress={()=>this.CambiarSigno()}
+              >
+             <Text style={styles.estiloTextoBotonDecimal_Signo}>+/-</Text>
+           </TouchableOpacity>
+          </View>
+
+
+         <View style={styles.columnaBotones}>
+          {columnasNumeros[1]}
+        </View>
+         <View style={styles.columnaBotones}>
+         {columnasNumeros[2]}
+          <TouchableOpacity
+            style={styles.estiloBotonDecimal_Signo}
+            onPress={()=>this.ArrayNumerosDeOperacion('.')}
+           >
+            <Text style={styles.estiloTextoBotonDecimal_Signo}>.0</Text>
+          </TouchableOpacity>
           </View>
-        </ScrollView>
-      </SafeAreaView>
-    </>
-  );
-};
+        
+
+         <View style={styles.columnaBotones}>
+
+           {ops}
+
+          <TouchableOpacity
+            style={styles.estiloBotonOperacion}
+            onPress={()=>this.ResultadoOperacion(0)}
+           >
+            <Text style={styles.estiloTextoBotonIgual}>=</Text>
+            
+          </TouchableOpacity>
+
+        </View>
+       
+
+
+      </View>
+
+    </View>
+  )
+}}
+
 
 const styles = StyleSheet.create({
-  scrollView: {
-    backgroundColor: Colors.lighter,
+  
+  container:{
+    flex: 1,
   },
-  engine: {
-    position: 'absolute',
-    right: 0,
+
+  pantallaCalculadora:{
+    flex: 0.30,
+    backgroundColor: '#19191b',
   },
-  body: {
-    backgroundColor: Colors.white,
+
+  divisorPantallaBotones:{
+    flex: 0.12,
+    backgroundColor: '#5f5e61',
+    flexDirection:"row",
+    justifyContent: "space-around",
+    alignItems:"center"
   },
-  sectionContainer: {
-    marginTop: 32,
-    paddingHorizontal: 24,
+
+  containerBotones:{
+    flex: 0.8,
+    backgroundColor: '#1D1F1E',
+    justifyContent: "space-around",
+    flexDirection: 'row',
+     },
+
+ 
+
+  columnaBotones:{
+    justifyContent: "space-around",
   },
-  sectionTitle: {
-    fontSize: 24,
-    fontWeight: '600',
-    color: Colors.black,
+
+  estiloBoton:{
```

Comment:
```suggestion
  boton:{
```

no tiene mucho sentido llamar  `estiloBoton` a algo que esta dentro de un objeto `styles`(estilos), se infiere por contexto. Luego, al uso queda `styles.boton` en vez del redundante `sytles.estiloBoton`.